### PR TITLE
Feature/fx

### PR DESCRIPTION
**Pull request by plusbpm**

### CODE REVIEW

File: packages/ui/src/Fx/Fx.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+## Use
+
+```
+import { useFx } from '@rkta/ui';
+...
+
+const MyFunctionComponent = () => {
+  const { phone } = useMedia();
+  const [popoverProps, addEffect] = useFx(['popUp']);
+  return <Paper {...popoverProps} onClose={() => addEffect('fadeDown')} />
```
Comment (droganov) [inline packages/ui/src/Fx/Fx.mdx]: this is nonesense, give real example please

File: packages/ui/src/Fx/Popover.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React, { ReactNode } from 'react';
+
+import { Paper, Text } from '..';
+
+interface Props {
+  children: ReactNode;
+  onClose: () => void;
+}
+
+const Popover = ({ children, onClose, ...rest }: Props): ReactNode => {
```
Comment (droganov) [inline packages/ui/src/Fx/Popover.tsx]: What;s that for? is that a part of FX? please dlete it

**Review by droganov — CHANGES_REQUESTED** [review]

File: packages/ui/src/Fx/Popover.tsx (hunk 2 above)
Comment (plusbpm) [reply packages/ui/src/Fx/Popover.tsx]: deleted

File: packages/ui/src/Fx/Fx.mdx (hunk 1 above)
Comment (plusbpm) [reply packages/ui/src/Fx/Fx.mdx]: updated

**Pull request merged**